arch/arc/core/isr_wrapper.S

```diff
@@ -192,9 +192,9 @@ From FIRQ:
 
   o to coop
 
-    The address of the returning instruction from z_swap() is loaded in ilink and
-    the saved status32 in status32_p0, taking care to adjust the interrupt lock
-    state desired in status32_p0. The return value is put in r0.
+    The address of the returning instruction from z_arch_switch() is loaded
+    in ilink and the saved status32 in status32_p0, taking care to adjust the
+    interrupt lock state desired in status32_p0. The return value is put in r0.
```

Comment:
I believe the comment about adjusting interrupt lock isn't valid anymore, the lock adjustment is removed in the PR.

Reply:
Fixed